packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -154,6 +155,9 @@ export const Webchat = forwardRef((props, ref) => {
   const handleMenu = () => {
     menuIsOpened ? setMenuIsOpened(false) : setMenuIsOpened(true)
   }
+  const handleEmoji = () => {
+    emojiIsOpened ? setemojiIsOpened(false) : setemojiIsOpened(true)
```

Comment:
setemojiIsOpened => setEmojiIsOpened

where is setemojiIsOpened defined?

Reply:
here : const [emojiIsOpened, setemojiIsOpened] = useState(false)
It's a react hook state.

Reply:
thanks, please rename it to setEmojiIsOpened